packages/roosterjs-content-model/lib/modelToDom/handlers/handleTable.ts

```diff
@@ -29,10 +29,12 @@ export const handleTable: ContentModelBlockHandler<ContentModelTable> = (
 
         moveChildNodes(tableNode);
     } else {
+        const tableContainer = doc.createElement('div');
```

Comment:
Add comment to explain why we need this DIV.

Question: Do we want to apply this DIV to all tables, or only when cut/copy?

Reply:
We can keep it only for cut and paste.